src/Components/Form/Form.js

```diff
@@ -98,7 +97,11 @@ class Form extends Component {
             <option value="intermediate">Padawan</option>
             <option value="expert">The Force is Strong With This One...</option>
         </select>
-        <button type="button" onClick={this.checkInputs}>Enter, You Will</button>
+        <Link
+        //if checkinputs, go to movies, if not, sit on it 
+        to='/movies'>
+          <button onClick={this.checkInputs}>Enter, You Will</button>
+        </Link>
```

Comment:
I thought we found a way around having <Link> in the Form. We'll reconcile this in driver/navigator